app/inline-scripts/instant-globals.js

```diff
@@ -0,0 +1,30 @@
+(function () {
+	var gettersQueue = [];
+	var isListening = false;
+
+	window.onInstantGlobalsLoaded = function () {
+		window.document.dispatchEvent(new Event('instantGlobalsLoaded'));
+	};
+
+	window.getInstantGlobal = function (key, callback) {
+		function onInstantGlobalsLoaded() {
+			gettersQueue.forEach((getter) => {
+				getter.callback(window.Wikia.InstantGlobals[getter.key]);
+			});
+
+			gettersQueue = [];
+			document.removeEventListener('instantGlobalsLoaded');
```

Comment:
You could use `document.addEventListener('instantGlobalsLoaded', onInstantGlobalsLoaded, {once: true});` instead